packages/opencensus-core/src/stats/model/types.ts

```diff
@@ -0,0 +1,148 @@
+import {StatsExporter} from '../../exporters/types';
+
+/**
+ * Copyright 2018, OpenCensus Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export interface RegistredViews { [key: string]: View; }
+
+export interface Histogram {
```

Comment:
This should be `HistogramBucket`, and `type Histogram = HistogramBucket[]`.

Reply:
Ok. Changed.